app/src/main/java/com/android/voyageur/ui/trip/AddActivity.kt

```diff
@@ -360,11 +363,31 @@ fun AddActivityScreen(tripsViewModel: TripsViewModel, navigationActions: Navigat
                   }
                 }
 
+                fun formatPrice(price: Double): String {
+                  val symbols =
+                      DecimalFormatSymbols.getInstance(Locale.getDefault()).apply {
+                        groupingSeparator = '\''
+                      }
+                  val decimalFormat =
```

Comment:
Hi, I'm not sure what you're trying to do here. Can you give some examples and how they should be displayed in each case?

Reply:
Sure, for example 101053 should be displayed as 101'053 and 1503.76 should be displayed as 1'503.76.

Reply:
I see, thanks. From what I tested the ' doesn't get displayed.
I don't think that's a very important issue for now, the main thing we wanted was for the app not to crash when certain characters are introduced.
So you can remove this part of the code for now if you cannot make it work

Reply:
Ok, I guess I can remove this for now and I'll try to solve it in the next sprint.